frontend/public/components/utils/managed-by.tsx

```diff
@@ -19,38 +17,39 @@ import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/
 import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
 import { ResourceLink } from '.';
 
-export const ManagedByOperatorResourceLink: React.SFC<ManagerLinkProps> = (props) => {
-  const { csvName, namespace, owner, className } = props;
-  const name = owner.name;
-  const { group, version } = groupVersionFor(owner.apiVersion);
-  const kind = owner.kind;
-
-  const reference = referenceForGroupVersionKind(group)(version)(kind);
-  const { namespaced } = modelFor(reference);
-
-  const link = `/k8s/ns/${namespace}/${referenceForModel(
-    ClusterServiceVersionModel,
-  )}/${csvName}/${reference}/${name}`;
-
+export const ManagedByOperatorResourceLink: React.SFC<ManagerLinkProps> = ({
+  csvName,
+  namespace,
+  owner,
+  className,
+}) => {
+  const csvGroupVersionKind = referenceForModel(ClusterServiceVersionModel);
+  const ownerGroupVersionKind = referenceForOwnerRef(owner);
+  const ownerIsCSV = ownerGroupVersionKind === csvGroupVersionKind;
```

Comment:
I think we just want to check group and kind here. We don't care about the version... It's the same resource regardless of the API version referenced.